pandas/core/nanops.py

```diff
@@ -637,7 +619,7 @@ def _maybe_null_out(result, axis, mask):
             else:
                 result = result.astype('f8')
             result[null_mask] = np.nan
-    else:
+    elif result is not tslib.NaT:
```

Comment:
If you take a ts and take a view as an int and then reduce, I think you still want `nan`. The only reason that you would want a `NaT` is that the dtype of the sequence being reduced is `datetime64`.